File: core/Core/Models/TransactionFee.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class TransactionFee
{
    public int Id { get; set; }

    public int? TransactionId { get; set; }

    public int? FeeId { get; set; }

    public decimal? Amount { get; set; }

    public virtual FeeType? Fee { get; set; }

    public virtual Transaction? Transaction { get; set; }
}

File: analytic/domain/transaction/transaction_fee.py
from decimal import Decimal


class TransactionFee:
    def __init__(self,
                 id: int,
                 transaction_id: int | None,
                 fee_id: int | None,
                 amount: Decimal | None):
        self.id = id
        self.transaction_id = transaction_id
        self.fee_id = fee_id
        self.amount = amount
